src/services/document-service.js

```diff
@@ -104,6 +104,19 @@ class DocumentService {
     return this.documentStore.getDocumentTagsMatchingText(searchString);
   }
 
+  async findDocumentsMetadata(query) {
+    const sanitizedQuery = query.trim();
+
+    const queryConditions = [
+      { archived: false },
+      { title: { $regex: `^${sanitizedQuery}`, $options: 'i' } }
```

Comment:
A second thought: Have you tried removing the `^` from the regex, so users can search for any part of the title (it's hard to remember sometimes how the title exactly starts, I guess)? If that would work too, I'd prefer that.

Reply:
I thought that for users that actually type the title from the beginning it would be strange to not find it on the top. 
But I updated as you suggested and we can see what feedback we get (if any).

Reply:
:+1: